src/Orchard/Localization/Services/DefaultLocalizedStringManager.cs

```diff
@@ -254,20 +254,28 @@ public class DefaultLocalizedStringManager : ILocalizedStringManager {
             }
         }
 
+        private static string TrimQuote(string str) {
+            if (str.StartsWith("\"") && str.EndsWith("\"")) {

```

Comment:
Good point. I see that the `DefaultLocalizedStringManager` doesn't have any tests and it is coupled with a lot of interfaces, so I'll have to refactor it a little bit. I'll update the PR soon.